controller/src/main/java/io/pravega/controller/server/rpc/grpc/v1/ControllerServiceImpl.java

```diff
@@ -358,7 +381,8 @@ public void checkTransactionState(TxnRequest request, StreamObserver<TxnState> r
     public void createScope(ScopeInfo request, StreamObserver<CreateScopeStatus> responseObserver) {
         RequestTag requestTag = requestTracker.initializeAndTrackRequestTag(requestIdGenerator.get(), "createScope", request.getScope());
         log.info(requestTag.getRequestId(), "createScope called for scope {}.", request.getScope());
-        authenticateExecuteAndProcessResults(() -> this.authHelper.checkAuthorization(request.getScope(), AuthHandler.Permissions.READ_UPDATE),
+        authenticateExecuteAndProcessResults(() -> this.authHelper.checkAuthorization(
+                AuthResourceRepresentation.ofScopes(), AuthHandler.Permissions.READ_UPDATE),
```

Comment:
Another one of the semantic changes, the authorization to create a scope depends on the permission over `ofScopes` which is `/`, right?

Reply:
Yes, that's right. We now use the parent resource representation for authorization of create operations such as this one.